### ISSUE TITLE
ExactTarget for email: Transition planning and timelines

### ISSUE THREAD

**Issue by valianttry** [mozilla/donate.mozilla.org#529]
@adamlofting this might be the wrong repo for this, please let me know if so and I'll fix.

Questions are starting to come up as we approach heavy fundraising and email season :rotating_light: 

General questions:
* When will we be able to launch emails from ExactTarget?
* What audience segmentation will be possible immediately? (e.g. just major interest groups, or will we be able to send email based on behavior / past actions, like signed a petition or clicked an email)?

Specific questions about email appeals for $$:
* Will we be able to segment based on donor vs. nondonor? 
* Will we be able to personalize email content based on data currently stored in BSD (e.g. previous donation amount; monthly donor; etc)?

cc @HPaulJohnson @KevZawacki @edrushka

**Comment by adamlofting** [comment]
This repo is fine for this :+1: 

> When will we be able to launch emails from ExactTarget?

* *Technically*, right now. Email templates are setup, and our 1.6 million subscribers are imported.
* *Practically*, in the next couple of weeks. I need to arrange training, update the teams who send email, and setup a final piece of middleware to keep BSD in sync.

> What audience segmentation will be possible immediately? (e.g. just major interest groups, or will we be able to send email based on behavior / past actions, like signed a petition or clicked an email)?

* Immediately, there is limited segmentation in ExactTarget until we have full integration with Salesforce (but there are plans in place to support end of year I'll note below).
* In ET right now you could select on 
 * Country (where available)
 * Language (EN, or DE for the new list)
 * High level interest in specific MoFo programmes (Webmaker, Learning Networks,  Mozfest, Maker Party)

**$ EOY**

Once Salesforce integration is complete, selection will allow all donor history plus engagement with the 300+ constituent groups in ET, but I'll answer the rest of these questions on the assumption we're going to start doing EOY work before Salesforce integration is complete...

> Will we be able to segment based on donor vs. nondonor? 

* Yes, to do this we'll setup a temporary data table in Exact Target with a record of previous giving by email address
 * This will give us a better donor history than we currenty have in BSD as it can include PayPal donations as well
 * This is going to need a little bit of manual work between @jbuck and I, but much of the ground work is already done

> Will we be able to personalize email content based on data currently stored in BSD (e.g. previous donation amount; monthly donor; etc)?

* Yes, this will be dependent on the same temporary donor history table we will setup in ET. If we can decide on the type of segmentation / personalisation you're going to want up front, we can make sure that data meets those needs.
 * I'm currently assuming: previous_highest_gift, number_of_gifts, last_donation_date, is_regular_donor

**Comment by valianttry** [comment]
@adamlofting all ^ sounds absolutely fantastic. I presume the work will pay for itself and then some. :hand:
